sig-contributor-experience/contribex-review-guidelines-on-test-infra.md

```diff
@@ -0,0 +1,123 @@
+
+---
+
+## Recommended Areas for ContribEx Review
+
+### Changes to [`config/prow/plugins.yaml`](https://github.com/kubernetes/test-infra/blob/52fb24b56def58e3cc36b10802aa6dea230584a7/config/prow/plugins.yaml)
+
+#### Plugins under ContribEx Review
+
+Following plugins fall under ContribEx review:
+
+- trigger
+- owners
+- approve
+- help
+- size
+- label
+- blockades
+- slack
+- welcome
+- require_matching_label
+
+#### Plugins Configured for Specific Orgs/Repos
+
+**Organizations:**
+- kubernetes
+- kubernetes-client
+- kubernetes-sigs
+- etcd-io
+
+**Repositories:**
+- kubernetes/kubernetes
+- kubernetes/community
+- kubernetes/org
+- kubernetes/publishing-bot
+- kubernetes-sigs/contributor-playground
+
+#### External Plugins Configured for Specific Orgs/Repos
+
+**Organizations:**
+- kubernetes
+- kubernetes-sigs
+- kubernetes-csi
+- kubernetes-client
+
+---
+
+### Changes to [`config/prow/config.yaml`](https://github.com/kubernetes/test-infra/blob/52fb24b56def58e3cc36b10802aa6dea230584a7/config/prow/config.yaml)
+
+- [slack_reporter_configs](https://github.com/kubernetes/test-infra/blob/52fb24b56def58e3cc36b10802aa6dea230584a7/config/prow/config.yaml#L215-L226)
+- [branch-protection](https://github.com/kubernetes/test-infra/blob/52fb24b56def58e3cc36b10802aa6dea230584a7/config/prow/config.yaml#L228-L636)
+- [tide](https://github.com/kubernetes/test-infra/blob/master/config/prow/config.yaml#L638-L844)
+- [github_reporter](https://github.com/kubernetes/test-infra/blob/52fb24b56def58e3cc36b10802aa6dea230584a7/config/prow/config.yaml#L850-L853)
+
+---
+
+### Changes to [`config/prow/OWNERS`](https://github.com/kubernetes/test-infra/blob/52fb24b56def58e3cc36b10802aa6dea230584a7/config/prow/OWNERS#L1) file
+
+Any changes to the reviewers/approvers or filters listed in the `config/prow/OWNERS` file.
+
+---
+
+### Changes to Prow CI Jobs and Their OWNERS Files
+
+Changes to the definition of the following Prow CI jobs files as well as their respective OWNERS files, owned directly or indirectly by SIG ContribEx:
+
+- All files under [kubernetes/community/](https://github.com/kubernetes/test-infra/tree/master/config/jobs/kubernetes/community) path
+- All files under [kubernetes/org/](https://github.com/kubernetes/test-infra/tree/master/config/jobs/kubernetes/org) path
+- [kubernetes/sig-k8s-infra/trusted/sig-contribex-k8s-triage-robot.yaml](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-k8s-infra/trusted/sig-contribex-k8s-triage-robot.yaml)
+- [kubernetes/sig-k8s-infra/trusted/sig-contribex-peribolos.yaml](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-k8s-infra/trusted/sig-contribex-peribolos.yaml)
+- [kubernetes/sig-k8s-infra/trusted/sig-k8s-infra-groups.yaml](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-k8s-infra/trusted/sig-k8s-infra-groups.yaml)
```

Comment:
Erm, this is SIG K8s Infra? I wouldn't expect changes to keeping this automation running to require input from other sigs, this is a technical implementation.